lib/rubocop/cop/rails/dynamic_find_by.rb

```diff
@@ -57,6 +61,33 @@ def autocorrect(node)
 
         private
 
+        def allowed?(node)
```

Comment:
Maybe a colorful name.

```suggestion
        def allowed_invocation?(node)
```